spec/requests/media/odk_response_spec.rb

```diff
@@ -207,4 +255,25 @@ def prepare_odk_media_upload_fixture(filename, form)
       formver: [form.number],
       itemcode: ODK::DecoratorFactory.decorate_collection(form.preordered_items).map(&:odk_code))
   end
+
+  def post_submission(submission_file, image_name, image, incomplete)
```

Comment:
same suggestion here: make `incomplete` a named method param (the rest are fine since it's quite obvious what's being passed in)

Reply:
done